### PR TITLE
feat(Assessments): add post /assessments/assess endpoint

### PR DESCRIPTION
**Pull request by julianalucena**
```
curl --request POST \
  --url localhost:3000/assessments/assess \
  --header 'Accept: application/json' \
  --header 'Authorization: YXBwX2lkOnRva2Vu\n' \
  --header 'Incognia-Installation-ID: some-installation' \
  --header 'Content-Type: application/json' \
  --data '{
    "account_id": "791ebdad-755a-4b55-8f2a-afab4ee07d9e"
}'

[{"api_name":"Onboarding","timestamp":"2022-10-17T10:20:59.540-03:00"},{"api_name":"Login","timestamp":"2022-10-17T10:21:00.356-03:00"}]%
```

### CODE REVIEW

File: app/services/base_entity.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class BaseEntity
```
Comment (ottony) [inline app/services/base_entity.rb:1]: This can be removed, right?

File: spec/requests/assessments_spec.rb [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        expect(response.body).to eq(serializable_assessments.to_json)
+      end
+
+      it_behaves_like 'handle Incognia API errors' do
+        let(:service) { form }
+        let(:method) { :submit }
+      end
+    end
+
+    context 'when validations fails' do
```
Comment (ottony) [inline spec/requests/assessments_spec.rb:66]: What happens when can't find the user?

https://github.com/inloco/incognia-demo-api-ruby/pull/29/files#diff-cf4c5fba34af4eeec826dad9c4c9380440487102292e3f91bf40801086e06ffaR4

**Review by ottony — APPROVED** [review]

File: app/services/base_entity.rb (hunk 1 above)
Comment (julianalucena) [reply app/services/base_entity.rb:1]: Actually no. It was removed in the next PR.

File: spec/requests/assessments_spec.rb (hunk 2 above)
Comment (julianalucena) [reply spec/requests/assessments_spec.rb:66]: Holly shit. I thought it was implemented to every child of `ApplicationController`.

I'm gonna open a new PR with some error-handling adjustments. :praise:

Comment (julianalucena) [reply spec/requests/assessments_spec.rb:66]: https://github.com/inloco/incognia-demo-api-ruby/pull/31/files

**Review by igorbpsimoes — APPROVED** [review]

**Pull request merged**